.github/workflows/i18n.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         m2-cache-key: 'i18n'
     - run: sudo apt install gettext
 
-    - run: ./bin/i18n/update-translation-template
-      name: Check i18n tags/make sure template can be built
-    - run: ./bin/i18n/build-translation-resources
-      name: Verify i18n translations (.po files)
+    - name: Check i18n tags/make sure template can be built
+      run: ./bin/i18n/update-translation-template
+    - name: Verify i18n translations (.po files)
+      run: ./bin/i18n/build-translation-resources
```

Comment:
This is purely stylistic change.
Maybe it's just me, but it always feels more natural to first read the `name` (to understand what are we doing), and only then to `run` something.

Reply:
I mostly see people list `name` first before `run`. So the change makes perfectly sense to me.